Assets/Scripts/Player/PlayerUi.cs

```diff
@@ -30,19 +28,23 @@ public GameObject getSelectedSegment()
 
     public void select()
     {
+        //Turns on the UI
         ui.gameObject.SetActive(true);
         updatePos();
-        ui.gameObject.SetActive(true);
+        //Turns off the UI if button no longer held
         if (!state.Select)
         {
             ui.gameObject.SetActive(false);
         }
+        //Points the UI to the main camera
+        ui.transform.LookAt(transform.position + Camera.main.transform.rotation * Vector3.forward,
+            Camera.main.transform.rotation * Vector3.up);
```

Comment:
Needs rotate every frame. so needs a proper reference to camera which it uses every update.